### PR TITLE
[ODS-4565] Update SwaggerUI Powershell Installer for .NET core apps

### PR DESCRIPTION
**Pull request by BhanuprakashThunga**

### CODE REVIEW

File: Scripts/NuGet/EdFi.Installer.SwaggerUI/Install-EdFiOdsSwaggerUI.psm1 [hunk 1]
```diff
@@ -302,15 +277,18 @@ function Invoke-TransformWebConfigAppSettings {
...
             $Config.PrePopulatedSecret = [string]::Empty
         }
-
+        
         $appSettings = @{
             "swagger.webApiMetadataUrl" = $Config.WebApiMetadataUrl
             "swagger.webApiVersionUrl" = $Config.WebApiVersionUrl
             "swagger.prepopulatedKey" = $Config.PrePopulatedKey
             "swagger.prepopulatedSecret" = $Config.PrePopulatedSecret
         }
```
Comment (tbmeador-dlp) [inline Scripts/NuGet/EdFi.Installer.SwaggerUI/Install-EdFiOdsSwaggerUI.psm1:286]: we need to update this to match the new format:
```suggestion
        $appSettings = @{
            WebApiVersionUrl = $Config.WebApiMetadataUrl
            SwaggerUIOptions = @{
                OAuthConfigObject = @{
                    ClientId     = $Config.PrePopulatedKey
                    ClientSecret = $Config.PrePopulatedSecret
                }
            }
        }
```

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
once installer are you able to open the website?

**Comment by tbmeador-dlp** [comment]
you can use package version `5.1.0-b12868` to test this one.

File: Scripts/NuGet/EdFi.Installer.SwaggerUI/Install-EdFiOdsSwaggerUI.psm1 [hunk 2]
```diff
@@ -302,15 +277,21 @@ function Invoke-TransformWebConfigAppSettings {
...
             $Config.PrePopulatedSecret = [string]::Empty
         }
-
+        
         $appSettings = @{
-            "swagger.webApiMetadataUrl" = $Config.WebApiMetadataUrl
-            "swagger.webApiVersionUrl" = $Config.WebApiVersionUrl
-            "swagger.prepopulatedKey" = $Config.PrePopulatedKey
-            "swagger.prepopulatedSecret" = $Config.PrePopulatedSecret
+            WebApiVersionUrl = $Config.WebApiMetadataUrl
```
Comment (tbmeador-dlp) [inline Scripts/NuGet/EdFi.Installer.SwaggerUI/Install-EdFiOdsSwaggerUI.psm1:282]:
```suggestion
            WebApiVersionUrl = $Config.WebApiVersionUrl
```

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
A few small things that I missed on the last round:

Lets remove the mandatory attribute here:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/bc57c003aa7efcd168ecf47bb5f97980786f88eb/Scripts/NuGet/EdFi.Installer.SwaggerUI/Install-EdFiOdsSwaggerUI.psm1#L84-L86
and we can mark it as obsolete:
```powershell
[Obsolete("This parameter is deprecated, and will be removed in the near future.")]
```

We should also update the test.ps1 and remove this parameter.

**Review by jamessmckay — APPROVED** [review]

**Pull request merged**